<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordAndPlay</class>
 <widget class="QMainWindow" name="RecordAndPlay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>音频引擎版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>播放音量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>录音设备：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPlayoutDevice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRecordingDevice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSpeakerVolumeValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>录音音量：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlayout">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始播放</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVersion">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMicrophoneVolumeValue">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRecording">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始录音</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMicrophoneVolume">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSpeakerVolume">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>371</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>播放设备：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
